tests/e2e/util.go

```diff
@@ -2126,3 +2126,78 @@ func getPersistentVolumeClaimSpecForFileShare(namespace string, labels map[strin
 	pvc.Spec.AccessModes = []v1.PersistentVolumeAccessMode{accessMode}
 	return pvc
 }
+
+// waitForPVCToComplete waits for PVC to complete PVC resize. It waits till the condition "FileSystemResizePending" is gone from the PVC condetion status
+func waitForPVCToCompleteResizing(client clientset.Interface, namespace string, pvcName string, poll time.Duration, timeout time.Duration) error {
```

Comment:
Same comments as https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/601 for the util methods

Reply:
this is removed